pkg/projectfile/projectfile.go

```diff
@@ -972,9 +975,46 @@ func createCustom(params *CreateParams, lang language.Language) (*Project, error
 		return nil, err
 	}
 
+	if params.Cache != "" {
+		createErr := createCacheFile(params.Directory, params.Cache)
```

Comment:
Can we name this `createHostFile()`? Cache is just the first usage of it. But I foresee we may want to use this for other things in the future.